contributions/catalog/catalog_rest_service.py

```diff
@@ -58,9 +69,11 @@ def callback():
     """ Step 3: Retrieving an access token.
     """
     # print("Step 3: Retrieving an access token")
-    github = OAuth2Session(cfg.client_id, state=session['oauth_state'])
-    token = github.fetch_token(cfg.token_url, client_secret=cfg.client_secret,
+    github = OAuth2Session(client_id, state=session['oauth_state'])
+    token = github.fetch_token(token_url, client_secret=client_secret,
                                authorization_response=request.url)
+
+    print(token)
```

Comment:
remove this after all the test of the code is done? for the release?

Reply:
I removed all print statements.